test/kpi-tokens-manager/KpiTokensManagerUpdateTemplateSpecificationTest.sol

```diff
@@ -43,4 +44,105 @@ contract KpiTokensManagerUpdateTemplateSpecificationTest is BaseTestSetup {
         assertEq(_template.id, _templateId);
         assertEq(_template.specification, _newSpecification);
     }
+
+    function testSuccessPastVersion() external {
+        assertEq(kpiTokensManager.templatesAmount(), 1);
+
+        string memory _oldSpecification = "a";
+        kpiTokensManager.addTemplate(address(2), _oldSpecification);
+        uint256 _templateId = kpiTokensManager.templatesAmount();
+        Template memory _template = kpiTokensManager.template(_templateId);
+        assertEq(_template.id, _templateId);
+        assertEq(_template.version, 1);
+        assertEq(_template.specification, _oldSpecification);
+
+        // add an additional template to the set by upgrading
+        string memory _upgradedSpecification = "b";
+        kpiTokensManager.upgradeTemplate(
+            _templateId,
+            address(3),
+            _upgradedSpecification
+        );
+        // templates amount is 2 because the test templates manager starts with
+        // one template being present
+        assertEq(kpiTokensManager.templatesAmount(), 2);
+        Template memory _upgradedTemplate = kpiTokensManager.template(
+            _templateId
```

Comment:
We should get version `2` of the template here, right? Or does it get the latest by default?

```
kpiTokensManager.template(
    _templateId,
    2
)
```

Reply:
Absolutely, that function gets the latest version by default.